spec/graphql/resolvers/invoice_resolver_spec.rb

```diff
@@ -93,6 +95,28 @@
     expect(group['value']).to eq('usa')
   end
 
+  context 'when invoice has credit notes' do
+    before do
+      create(:credit_note, invoice: invoice)
+    end
+
+    it 'returns an error' do
+      result = execute_graphql(
+      current_user: membership.user,
+      current_organization: organization,
+      query: query,
+      variables: { id: invoice.id },
+    )
+    
+
+    data = result['data']['invoice']
+
+    aggregate_failures do
+      expect(data['hasCreditNotes']).to be_truthy
+    end
+    end
```

Comment:
Something is weird around here :p

Reply:
got spammed by the linter 😄 should definitely investigate why it don't work on mu IDE 😓